src/components/CertificateTemplates/tlds/sg/edu/ntu/common/functions.js

```diff
@@ -0,0 +1,34 @@
+export const formatDate = dateString => {
+  if (!dateString) return null;
+  const date = new Date(dateString);
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  ];
+  const month = date.getMonth();
+  const year = date.getUTCFullYear();
+  const day = date.getDate();
+  return `${day} ${months[month]} ${year}`;
+};
+
+export const formatNRIC = nricFin => {
```

Comment:
i didn't find a nric field in your cert, but if you do use a field for nric/fin please use separate key names for nric and fin
e.g

```
"recipient": {
  "nric": "SxxxxxxxY"
}

or

"recipient": {
  "fin": "FxxxxxxxY",
}
```

Reply:
ok, we are not using nric, have removed that function